src/main.ts

```diff
@@ -20,6 +21,8 @@ async function bootstrap() {
     .build();
 
   const document = SwaggerModule.createDocument(app, options);
+  fs.writeFileSync('./swagger.json', JSON.stringify(document, null, 2));
```

Comment:
@asad-mlbd do you want to move this file generation to a separate step?
If I remember it right, `serverless` environments don't give the app a write access to the file system.
So our backend will not be able to run in that environment due to that constraint. :thinking: